resources/install.rb

```diff
@@ -45,6 +45,8 @@
   node.run_state['haproxy']['conf_template_source'][new_resource.config_file] = new_resource.conf_template_source
   node.run_state['haproxy']['conf_cookbook'][new_resource.config_file] = new_resource.conf_cookbook
 
+  normalize_build_flag = ->(use_value) { use_value.to_s == '1' ? '1' : '' }
```

Comment:
This would be better of in a helper method in https://github.com/sous-chefs/haproxy/blob/master/libraries/helpers.rb sow we can make some unit tests run against it.

Reply:
If we test this on other haproxy versions and it is expecting `''` or `1`, then perhaps we should just change the allowed values for those properties in the resource?